src/entrypoint.scss

```diff
@@ -3,17 +3,17 @@
 
 // This adds the background
 html::after {
-  content: "";
-  background: url(images/rwb.svg) no-repeat center center fixed;
-  -webkit-background-size: cover;
+  background: url('images/rwb.svg') no-repeat center center fixed;
```

Comment:
curious about the above -- what does adding this to the `::after` do different from adding it just to `html` (or some top-level child div of `html`)?